index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="reset.css">
+<link rel="stylesheet" href="main.css">
+   <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <meta charset="UTF-8">
+    <title>Apple</title>
+</head>
+<body>
+   <main>
+    
+ 
+        <nav>
+            <ul>
+                <li> <a href="#"><div class="logo"></div> </a></li>
+                <li> <a href="#"> Store </a></li>
+                <li> <a href="#"> Mac </a></li>
+                <li> <a href="#"> iPod </a></li>
+                <li> <a href="#"> iPhone </a></li>
+                <li> <a href="#"> iPad </a></li>
+                <li> <a href="#"> iTunes </a></li>
+                <li> <a href="#"> Support </a></li>
+                <li> 
+                <form action="">
```

Comment:
You have just one error on validator. i trust you fix it, move to the next project afterward.
Note: you can perfectly use an input tag without wrapping it with a form tag.